iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/ThreeWindingsTransformerXml.java

```diff
@@ -22,6 +22,11 @@
 
     static final String ROOT_ELEMENT_NAME = "threeWindingsTransformer";
 
+    private static final String RATIO_TAP_CHANGER_1 = "ratioTapChanger1";
```

Comment:
Maybe you should also introduce RATIO_TAP_CHANGER_2 and RATIO_TAP_CHANGER_3 even there is no duplication yet